### PR TITLE
update copy for list of requirements

### PR DESCRIPTION
**Pull request by Ajiva-D**

### CODE REVIEW

**Review by arne-z — CHANGES_REQUESTED** [review]

File: packages/grants/pages/apply/index.tsx [hunk 1]
```diff
@@ -16,41 +16,53 @@ import styled from "styled-components";
...
+      html: `<div class="text-gray-500">
+		<p>	In order to be considered as a potential beneficiary, your organisation must aid social impact efforts in one of the following four domains:</p>
+			<ol class="pl-4 list-decimal">
+				<li>Environment</li>
+		<li>	Free and open-source software</li>
+			<li>Education</li>
+		<li>	Inequality</li>
+			</ol>
+			<p>Fill out the <a href="/" class="text-blue-600 font-semibold">Popcorn Foundation Social Impact Questionnaire</a></p>
+			</div>`,
```
[the remark this answers is not in the document]
Comment (Ajiva-D) [reply packages/grants/pages/apply/index.tsx]: Done

File: packages/grants/pages/apply/index.tsx [hunk 2]
```diff
@@ -16,41 +16,67 @@ import styled from "styled-components";
 interface RequirementsObject {
   icon: ReactElement;
   title: string;
-  content: string;
+  content?: string;
+  html?: JSX.Element;
```
Comment (arne-z) [inline packages/grants/pages/apply/index.tsx:20]: would rename this. I think "children" should work.

File: packages/grants/pages/apply/index.tsx [hunk 3]
```diff
@@ -16,41 +16,67 @@ import styled from "styled-components";
...
     {
       icon: <ClipboardCheckIcon />,
-      title: "Basic Information",
-      content:
-        "Here is the basic information you would need to complete your application: the organization name, the title of your application, the proposal category, the organization's mission statement, the official email address, and all links to your social accounts.",
+      title: "Fit at least into one of the four categories",
+      html: (
+        <div className="text-gray-500">
+          <p>
+            {" "}
```
Comment (arne-z) [inline packages/grants/pages/apply/index.tsx:31]:
```suggestion
```

File: packages/grants/pages/apply/index.tsx [hunk 4]
```diff
@@ -16,41 +16,67 @@ import styled from "styled-components";
...
     },
     {
-      icon: <SupportIcon />,
-      title: "Proof of Ownership",
-      content:
-        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Purus rhoncus, blandit at porta viverra. Nunc, convallis aenean sed faucibus quis facilisis tincidunt morbi sapien. Hendrerit ",
+      icon: <PhotographIcon />,
+      title: "Visual Presentation",
+      content: `A logo image file with a maximum size of 400KB. A square (1:1) PNG file with a transparent background is recommended.
+				o Two image files that best represents your work with a definition of minimum 300dpi, one in portrait and one in landscape orientation.`,
```
Comment (arne-z) [inline packages/grants/pages/apply/index.tsx:54]: What's this: o Two image files...
Should this be a bullet point or did and extra o sneak in?

**Review by arne-z — CHANGES_REQUESTED** [review]

**Review by arne-z — APPROVED** [review]

**Pull request merged**